docs/changelog.rst

```diff
@@ -123,6 +129,14 @@ Cleanups
 - Catch errors when loading :confval:`logging/config_file`.
   (Fixes: :issue:`1320`)
 
+Audio
+-----
+
+- **Breaking:** The audio scanner now returns ISO-8601 formatted strings
```

Comment:
mopidy-local-sqlite doesn't use the audio scanner directly, but only uses the `Track` as passed by the local library interface.
mopidy-local-images *does* use the tags from the audio scanner, but only looks for `image` and `preview-image` tags.
So there shouldn't be any issues.